### PR TITLE
Fix some things

### PR DESCRIPTION
**Pull request by britaumich**
Remove all mentions of admin_user
Add [validate uniqueness] to all complementary tables
Make authorization for device's new and destroy methods false
Remove comments, an unneeded file and fix some formats

LegacyOsRecords & SensitiveDataSystems: 

- make device validation text red
- replace turbo_stream (that flashes device_class's messages) with render (because _form.html.erb is now [turbo: "false"]), and add these messages to <object>.errors
Replace this:
```
flash.now[:alert] = device_class.message
render turbo_stream: turbo_stream.update("flash", partial: "layouts/notification")
```
with this:
```
@legacy_os_record.errors.add(:device, device_class.message)
render :new
```

### CODE REVIEW

File: app/views/legacy_os_records/_form.html.erb [hunk 1]
```diff
@@ -215,7 +215,7 @@
             <div class="w-1/3 pr-4">
               <%= form.label :serial %>
               <%= device_attributes.text_field :serial, :"data-legacyos-target" => "serial" %>
-              <span class="device-error--hide" data-legacyos-target="serial_error">
+              <span class="device-error--hide text-red-500" data-legacyos-target="serial_error">
```
Comment (rsmoke) [inline app/views/legacy_os_records/_form.html.erb:218]: change this to text-red-700

File: app/views/sensitive_data_systems/_form.html.erb [hunk 2]
```diff
@@ -131,7 +131,7 @@
           <div class="w-1/3 pr-4">
             <%= form.label :serial %>
             <%= device_attributes.text_field :serial, :"data-sensitiveds-target" => "serial" %>
-            <span class="device-error--hide" data-sensitiveds-target="serial_error">
+            <span class="device-error--hide text-red-500" data-sensitiveds-target="serial_error">
```
Comment (rsmoke) [inline app/views/sensitive_data_systems/_form.html.erb:134]: change this to text-red-700

**Review by rsmoke — CHANGES_REQUESTED** [review]
we need to remove the coupling between the TDX Class and the controllers.
Fix the text color as noted

File: app/views/legacy_os_records/_form.html.erb (hunk 1 above)
Comment (britaumich) [reply app/views/legacy_os_records/_form.html.erb]: done

**Review by rsmoke — APPROVED** [review]
This will be need to refactored to use the flash function in turbo

**Pull request merged**